src/xcode/ENA/ENA/Source/Scenes/QRScanner/_tests_/CoronaTestsQRCodeParserTests.swift

```diff
@@ -401,20 +383,26 @@ class CoronaTestsQRCodeParserTests: CWATestCase {
 		XCTAssertNil(guid)
 	}
 
-	func testGIVEN_invalidPath_WHEN_extractAntigenPayload_THEN_isInvalid() {
+	func testGIVEN_invalidPath_WHEN_extractRapidPayload_THEN_isInvalid() {
 		// GIVEN
 		let parser = CoronaTestsQRCodeParser()
 
 		// WHEN
-		let result = parser.coronaTestQRCodeInformation(from: "https://s.coronawarn.app/?v=1#//?eyJ0aW1lc3RhbXAiOjE2MTgyMzM5NzksImd1aWQiOiIwQzg5MjItMEM4OTIyNjMtQTM0Qy00RjM1LTg5QUMtMTcyMzlBMzQ2QUZEIiwiZm4iOiJDYW1lcm9uIiwibG4iOiJIdWRzb24iLCJkb2IiOiIxOTkyLTA4LTA3In0")
+		let resultRAT = parser.coronaTestQRCodeInformation(from: "https://s.coronawarn.app/?v=1#//?eyJ0aW1lc3RhbXAiOjE2MTgyMzM5NzksImd1aWQiOiIwQzg5MjItMEM4OTIyNjMtQTM0Qy00RjM1LTg5QUMtMTcyMzlBMzQ2QUZEIiwiZm4iOiJDYW1lcm9uIiwibG4iOiJIdWRzb24iLCJkb2IiOiIxOTkyLTA4LTA3In0")
 
 		// THEN
-		XCTAssertNil(result)
+		XCTAssertNil(resultRAT)
+		
+		let resultRapidPCR = parser.coronaTestQRCodeInformation(from: "https://p.coronawarn.app/?v=1#//?eyJ0aW1lc3RhbXAiOjE2MTgyMzM5NzksImd1aWQiOiIwQzg5MjItMEM4OTIyNjMtQTM0Qy00RjM1LTg5QUMtMTcyMzlBMzQ2QUZEIiwiZm4iOiJDYW1lcm9uIiwibG4iOiJIdWRzb24iLCJkb2IiOiIxOTkyLTA4LTA3In0")
+
+		// THEN
+		XCTAssertNil(resultRapidPCR)
+
 	}
 
-	func testAntigen_hashIsTooShort() {
+	func testRapid_hashIsTooShort() {
 		let invalidHash = "f1200d9650f1fd673d58f52811f98f1427fab40b4996e9c2d0da8b741446408"
-		let antigenTestInformation = AntigenTestQRCodeInformation.mock(hash: invalidHash)
+		let antigenTestInformation = RapidTestQRCodeInformation.mock(hash: invalidHash)
```

Comment:
```suggestion
		let rapidTestInformation = RapidTestQRCodeInformation.mock(hash: invalidHash)
```

Same for the ones below, just to keep the naming consistent